internal/pkg/cli/storage_init.go

```diff
@@ -21,24 +20,41 @@ import (
 )
 
 const (
-	dynamoDBStorageType = "DynamoDB"
-	s3StorageType       = "S3"
-)
-
-const (
-	s3BucketFriendlyText      = "S3 Bucket"
-	dynamoDBTableFriendlyText = "DynamoDB Table"
-)
+	dynamoDBStorageType     = "DynamoDB"
+	s3StorageType           = "S3"
+	rdsStorageType          = "RDS"
 
-const (
-	ddbKeyString = "key"
+	dynamoDBStorageTypeFriendly = "DynamoDB"
+	s3StorageTypeFriendly       = "S3"
+	rdsStorageTypeFriendly      = "RDS (Aurora Serverless)"
```

Comment:
Is Aurora the only RDS type we are going to support? If so, why not just name it as Aurora and hide RDS from users? This makes me think if there's other RDS we'll have.

Reply:
For now I think Aurora Serverless is the only one that we are going to support.

I think we should discuss this on Monday. I believe we want to somehow keep the RDS keyword because people are more interested in the concept of RDS than that of Aurora. There might be folks that don't know about Aurora but do know that they want RDS. Just my thought 👀